torchvision/transforms/functional_pil.py

```diff
@@ -261,7 +290,13 @@ def _parse_fill(fill, img, name="fillcolor"):
 
 
 @torch.jit.unused
-def affine(img, matrix, interpolation=0, fill=None):
+def affine(
+    img: Image.Image,
+    matrix: List[float],
+    interpolation: int = 0,
+    fill: Optional[Union[float, List[float], Tuple[float, ...]]] = 0,
```

Comment:
FYI strangely the tensor API defines this as float as well.

Reply:
Actually the tensor API defines it to be only `List[float]` and not all the three, again something which isn't aligned or probably has an issue.

Reply:
It's the union that is not supported. I just highlight that here the type (float vs int) is actually aligned. Which is weird cause we don't seem to keep things aligned. I probably agree that it's the tensor that needs changing though.

Reply:
I just verified Sequence is not supported by JIT. And yes Union too is not supported.

https://pytorch.org/docs/stable/jit_language_reference.html#supported-type